microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -223,6 +242,138 @@ def verify_kdumpcrash_large_memory_auto_size(
         self.is_auto = True
         self._kdump_test(node, log_path, log)
 
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. replace FRE bins with CHK bins and reboot the VM
+        2. Configure kdump and reboot VM
+        3. Generate Crash with hvdbg syscall and verify dump
+
+        The test expects the directory containing MSHV CHK binaries tar to be passed
+        in the mshv_chk_bin / mshv_chk_loader testcase variables.
+        """,
+        priority=2,
+        requirement=node_requirement(
+            node=schema.NodeSpace(
+                core_count=4, memory_mb=search_space.IntRange(min=8192)
+            ),
+        ),
+    )
+    def verify_mshv_crash(
+        self,
+        log: Logger,
+        node: Node,
+        variables: Dict[str, Any],
+        log_path: Path,
+    ) -> None:
+        chkbinpath = variables.get("mshv_chk_bin", "")
+        chkloaderpath = variables.get("mshv_chk_loader", "")
+        if not chkbinpath or not chkloaderpath:
+            raise SkippedException(
+                "Requires a path to MSHV binaries to be passed."
```

Comment:
`"Requires a path to MSHV binaries to be passed. "`

Reply:
Done